src/test/tcp/test_tcp.c

```diff
@@ -49,6 +61,37 @@ static void _mylog(const char* fileName, const int lineNum, const char* funcName
     fflush(stdout);
 }
 
+// create a System V queue with key based on a file queue name
+static int get_queue(const char* queuename) {
+    key_t key = ftok(queuename, 0);
```

Comment:
Doing the assignment inside the check gives better error messages on failures. e.g.
```
key_t key;
assert_true_errno((key = ftok(queuename, 0) != -1);
```
On failure will output the expression `(key = ftok(queuename, 0) != -1` instead of just `key != -1`